modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/application/filter/LiferayEventProcessorFilter.java

```diff
@@ -54,7 +53,7 @@ public void filter(ContainerRequestContext containerRequestContext) {
 		try {
 			HttpSession httpSession = _httpServletRequest.getSession(false);
 
-			httpSession.removeAttribute(Globals.LOCALE_KEY);
+			httpSession.removeAttribute(WebKeys.LOCALE);
```

Comment:
can you commit this in a different PR, we shouldn't mix up things in PR especially this that needs to be backported too.

Thank you!

Reply:
We need to commit this under the same PR. 

Globals.LOCALE_KEY is dependent on 'compileOnly group: "org.apache.struts", name: "struts-core", version: "1.3.10"`. 
We no longer use `compileOnly group: "org.apache.struts", name: "struts-core", version: "1.3.10"` in Master, 7.3.x, and 7.2.x.
We need to replace in the same PR.

See: https://github.com/liferay/liferay-portal/commit/a1db809bb052162dcb71fb218db47c1bb85a6789
and: https://github.com/liferay/liferay-portal/commit/0f6e75203fd4fd674807e92cfe011c1f560f2ea6

Reply:
I'm not 100% convinced, this was introduced later to those commits see https://github.com/marco-leo/liferay-portal-ee/pull/857 for explanation. I think we can drop the dependency from Struts, but I'm not convinced we should change the key to remove our locale, you can use the value from the struts constant.
Anyway I would reach out to tina to understand if her fix is still valid or we can accept your. 
I hope I was clear, reach out to me in case you need clarification.

Reply:
Hi @andrea-ale-sbarra , we no longer use `Globals.LOCALE_KEY` anywhere in the code. It was all replaced by `WebKeys.LOCALE`. Her fix was based on Master prior to the introduction of LPS-88361.

Reply:
@tinatian, could you confirm whether it is safe to drop your implementation?

Sincerely,
Brian

Reply:
@brianikim This was added to fix an cache issue, see the reason at https://github.com/marco-leo/liferay-portal-ee/pull/857#issue-623539488. If you are sure that the property key populated changed since then, you need to update the key, or else, please use hard code string so that you won't need to class import.

Reply:
@tinatian , LPS-88361 is not being tracked by JIRA since the migration (very unfortunate, I think all older LPS tickets did not get fully migrated). According to my investigation, LPS-88361 removed all usage of Globals.LOCALE_KEY and struts-core. The property key is populated by `WebKeys.LOCALE` now.

Reply:
From my understanding, the code change introduced by your commits no longer worked since the introduction of LPS-88361 because `Globals.LOCALE_KEY` was never set anywhere.

Reply:
@brianikim If that's the case, could you please verify if the original bug my change fixed still can be reproduced, if not, then you can remove all the logic I added to fix that bug.

Reply:
@andrea-ale-sbarra , @tinatian , original bug cannot be reproduced with my fix. We should be fine.